python/CNN_v2.0.0.py

```diff
@@ -5,25 +5,32 @@
 
 # Based on:
 # https://blog.keras.io/using-pre-trained-word-embeddings-in-a-keras-model.html
-
-# ### Load requirements and data
 import os
+MODEL_NAME = os.getenv('EXPERIMENT_NAME')
+DATADIR = os.getenv('DATADIR')
+print('algorithm running on data extracted from content store on {}'.format(DATADIR))
 
+# ### Load requirements and data
+from comet_ml import Experiment
 from tokenizing import load_tokenizer_from_file
 from algorithm_functions import f1, to_file, WeightedBinaryCrossEntropy
 
 import numpy as np
+import pandas as pd
 
 from keras.callbacks import EarlyStopping
 from keras.layers import (Embedding, Input, Dense, Dropout, 
                           Activation, Conv1D, MaxPooling1D, Flatten, concatenate, Reshape)
 from keras.models import Model, Sequential
-from sklearn.metrics import precision_recall_fscore_support, classification_report
+from sklearn.metrics import precision_recall_fscore_support, classification_report, f1_score
 
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+rcParams.update({'figure.autolayout': True})
+import json
 
-
-DATADIR = os.getenv('DATADIR')
-print('algorithm running on data extracted from content store on {}'.format(DATADIR))
+experiment = Experiment(api_key="ohCiHJ1pXoSLP46AeItiYm0ke",
```

Comment:
The api_key probably shouldn't be committed, it's probably best to pass it in as an environment variable. This can then be stored in the `.local.envrc` file.